go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -295,32 +294,10 @@ func (c *Controller) syncServices(key string) error {
 			// mark the vip as processed
 			vipsTracked = vipsTracked.Delete(virtualIPKey(vip, svcPort.Protocol))
 
-			// handle reject ACLs for services without endpoints
-			// eventually we have to remove this because it will
-			// be implemented by OVN
-			// https://github.com/ovn-org/ovn-kubernetes/pull/1887
-			rejectACLName := loadbalancer.GenerateACLNameForOVNCommand(lbID, ip, svcPort.Port)
-			aclID, err := acl.GetACLByName(rejectACLName)
-			if err != nil {
-				klog.Errorf("Error trying to get ACL for Service %s/%s: %v", name, namespace, err)
-			}
-			// if there is no ACL and there is no endpoints we add a new ACL
-			// if there is an ACL and we have endpoints we have to remove the ACL
-			// if there is no ACL and we have endpoints we don´t need to do anything
-			if len(eps) == 0 && len(aclID) == 0 {
+			// reject ACLs are handled by OVN
```

Comment:
not sure this comment is really necessary

Reply:
doesn´t harm , for future us or newcomers will be useful